### ISSUE TITLE
Issue: Error parsing is not optimal.

### ISSUE THREAD

**Issue by Sahil-Chhoker** [ishitaraina1807/Reading-Matters#1]
Error parsing on the signup page is very vague.
Issues: 
1. Does not tell us about the email if it is already used.
2. Does not tell us if the email is wrong.
3. Does not tell us if the account already exists.

**Comment by ishitaraina1807** [comment]
Hello sahil, you can start working in your issues. Thanks for raising these. When you make the pull request make sure you tag the issue number.

**Comment by Sahil-Chhoker** [comment]
Yeah i am okay to work on the issue, but i need some information about the layout of your project. You mind telling me where exactly are you doing the error parsing for email?

**Comment by ishitaraina1807** [comment]
Sure @Sahil-Chhoker. I has been done in Auth folder - for both signup and login. Clone and try to understand the project. If you face any issue, then please feel free to contact.

**Comment by Sahil-Chhoker** [comment]
I think you should provide some info about setting up the project, like am i supposed to make a .env file for all the api keys?
And how do i set up the firebase auth and firestore?

**Comment by ishitaraina1807** [comment]
I have provided the necessary details, kindly refer to youtube videos, setting up firebase isn't something I will teach you.

**Comment by Sahil-Chhoker** [comment]
Main issue wasn't the setting up of firebase though, i asked where do the api keys go.

**Comment by ishitaraina1807** [comment]
Navigate to firebase.js in component folder. API is not wrapped in .env file.

**Comment by Shahbaz898414** [comment]
can you assign me this issue?
